WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -634,6 +640,10 @@ private extension NotificationsViewController
             DDLogSwift.logError("Error refreshing Notification Row \(error)")
         }
     }
+
+    func setNeedsReloadResults() {
```

Comment:
Inspired by `setNeedsDisplay` + this way the ivar is only accessed directly within the `WPTableViewHandler Helpers` extension.